posts/share-cpp/index.qmd

```diff
@@ -14,12 +14,12 @@ format:
 
 Using the [{Rcpp}](https://www.rcpp.org/) package is the dominant method for linking the usability of R with the speed of C++, and can be used to write R packages that are fast and easy to use for both end-users and developers.
 
-From the point of view of developers, it's very easy to export R code such as functions and classes from an R(cpp) package, but there is little clear guidance on how to export the C++ code so that it can be shared with your other Rcpp packages.
+From the point of view of developers, it's very easy to export R code such as functions and classes from an R(cpp) package, but the guidance in the [Rcpp documentation](https://www.rdocumentation.org/packages/Rcpp/versions/1.0.10) is not very clear on how to export the C++ code so that it can be shared with your other Rcpp packages.
 
 Allowing C++ code to be shared can be very beneficial for the same reasons that sharing R code is --- packaging code is a reliable way to reuse it.
 
-Some widely used examples of this practice are the {RcppEigen}, {RcppArmadillo}, {RcppGSL}, and Boost Headers {BH} packages.
-Indeed, in the Epiverse-TRACE team, {RcppEigen} underpins the {finalsize} and upcoming {epidemics} packages.
+Some widely used examples of this practice are the [{RcppEigen}](https://cran.r-project.org/web/packages/RcppEigen/index.html), [{RcppArmadillo}](https://cran.r-project.org/web/packages/RcppArmadillo/index.html), [{RcppGSL}](https://cran.r-project.org/web/packages/RcppGSL/index.html), and [Boost Headers {BH}](https://cran.r-project.org/web/packages/BH/index.html) packages.
+Indeed, in the Epiverse-TRACE team, {RcppEigen} underpins the [{finalsize}](https://github.com/epiverse-trace/finalsize) and upcoming [{epidemics}](https://github.com/epiverse-trace/epidemics) packages.
 
 ## Two ways to share C++ code?
 
```

Comment:
Worth numbering items in lines 28-30?

Reply:
Yes, done in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/34/commits/5fc90a324aa144aba5fd0828d24bb1338f0c8e6f